torchx/mix.exs

```diff
@@ -6,7 +6,7 @@ defmodule Torchx.MixProject do
 
   @libtorch_version "1.9.1"
   @libtorch_target "cpu"
-  @libtorch_cache Path.join(__DIR__, "cache/libtorch-#{@libtorch_version}-#{@libtorch_target}")
+  @libtorch_nerves_url_base "https://github.com/cocoa-xu/elixir-nx-libtorch/releases/download/v#{@libtorch_version}"
```

Comment:
In the same vein as the Makefile comments, this seems to be too nerves-specific to be here.
Perhaps the logic can be reworked to just decide on whether to copy libtorch?

I also don't see why someone couldn't just download this nerves libtorch version and point LIBTORCH_DIR to ir directly, which bypasses the download process altogether.

Reply:
I think you can just revert the changes on mix.exs and point LIBTORCH_DIR to the unpacked .zip you created and things should work as expected (with the changes in the makefile)

Reply:
I'm totally okay with either way. The reason I'm doing this is just to simplify the install process. It's not necessary to do this in Nx. But since there is code for downloading LibTorch on `{:unix, :darwin}` and `{:unix, :linux}`, I thought it might be more complete if we add this.

Reply:
The main difference for me is that these are platforms directly supported by libtorch and by the BEAM directly. All other platforms can be supported by pointing directly to the correct LIBTORCH_DIR